src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,14 @@
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.AWSLambdaInstrumentationOptions
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.AWSLambdaInstrumentationOptions.AWSLambdaInstrumentationOptions() -> void
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.AWSLambdaInstrumentationOptions.DisableAwsXRayContextExtraction.get -> bool
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.AWSLambdaInstrumentationOptions.DisableAwsXRayContextExtraction.set -> void
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation.AWSLambdaWrapper
```

Comment:
What is the policy for moving to Shipped? For all kinds of "shippings", or do pre-releases (beta) not count?

Reply:
The entries are moved to Shipped from Unshipped when doing a stable release.